src/app/locales/zh_CN.ts

```diff
@@ -0,0 +1,444 @@
+  },
+  'cli v6 scripts': {
+    action: '使用新的 Angular CLI 命令更新 package.json 中的 `scripts`。现在所有的 CLI 命令都使用两个破折号来表示标志(flags)(例如 `ng build --prod --source-map`p)，以便与 POSIX 兼容。'
+  },
+  'Update to Angular v6': {
+    action: '将所有 Angular 框架软件包更新到 v6，并更新正确版本的 RxJS 和 TypeScript。<br/>`ng update @angular/core@6`<br/>更新后，TypeScript 和 RxJS 将更准确地作出类型检查，这可能会暴露现有程序中的类型错误。'
+  },
+  'forms v6': {
+    action: '现在在 Angular Forms 中，调用 `AbstractControl#markAsPending` 时，`AbstractControl#statusChanges` 将发出一个 `PENDING` 事件。如果要从 `statusChanges` 中筛选或检查事件，请确保在调用 `markAsPending` 时考虑新事件。'
+  },
+  'animations timing': {
+    action: '如果在禁用区域(disabled Zone)内从 `AnimationEvent` 使用 totalTime，它将不再报告时间 0。要检测动画事件是否报告了禁用动画，可以使用 `event.disabled` 属性代替。'
+  },
+  'ngModel on form control': {
+    action: '在 v6 中不推荐使用 ngModel 和 ngModelChange 事件与反应式表单指令(reactive form directives)，而在 v7 中则取消了这个不赞成(deprecated)。'
+  },
+  'ngModelChange order': {
+    action: 'ngModelChange 现在在其控件的 value/validity 更新后发出，而不是在更新之前，以更好地匹配预期。如果依赖于这些事件的顺序，则需要跟踪组件中的旧值。'
+  },
+  'Update Dependencies for v6': {
+    action: '将 Angular Material 更新到最新版本。<br/>`ng Update@Angular/Material`<br/>这也会自动迁移不推荐使用的 API。'
+  },
+  'strictPropertyInitializer': {
+    action: '如果您已将 TypeScript 配置为严格(如果在 `tsconfig.json` 文件中将 `strict` 设置为 `true`)，请更新 `tsconfig.json` 以禁用 `strictPropertyInitialization` 或将属性初始化从 `ngOnInit` 中移到构造函数。您可以在<a href="https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-7.html#strict-class-initialization"> TypeScript 2.7 发行说明</a>了解有关此的更多信息。'
+  },
+  'update to RxJS 6': {
+    action: '移除 RxJS 5 不推荐使用的功能使用 <a href="https://github.com/ReactiveX/rxjs-tslint" target="_blank">rxjs-tslint</a> 自动更新规则<br/>在大多数应用程序中，这需要运行以下两个命令:<br/>`npx rxjs-tslint`<br/>`rxjs-5-to-6-migrate -p src/tsconfig.app.json`'
+  },
+  'remove rxjs-compat': {
+    action: '一旦依赖更新到 RxJS 6，请移除 `rxjs-compat`.'
+  },
+  'use files instead of versionedFiles': {
+    action: '如果使用了 Angular Service worker，使用 `files` 替代 `versionedFiles`'
+  },
+  'TypeScript 3.1': {
+    action: 'Angular 现在使用 TypeScript 3.1，了解更多信息：<a href="https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-1.html" target="_blank">https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-1.html</a>'
+  },
+  'Node 10': {
+    action: 'Angular 现在增加了对 Node 10 的支持: <a href="https://nodejs.org/en/blog/release/v10.0.0/" target="_blank">https://nodejs.org/en/blog/release/v10.0.0/</a>'
+  },
+  'v7 update': {
+    action: '在终端中运行 `ng update @angular/cli@7 @angular/core@7` 更新 CLI 和核心框架到 v7。'
+  },
+  'v7 material update': {
+    action: '执行 `ng update @angular/material`，更新到 Angular 7、Angular 8、Angular 9、Angular 10，请对应执行 `ng update @angular/material@7`、`ng update @angular/material@8`、`ng update @angular/material@9`、`ng update @angular/material@10`'
```

Comment:
Angular 8, Angular 9, Angular 10 多了嗎?

Reply:
If you choose to upgrade across multiple versions and use Angular Material

![image](https://user-images.githubusercontent.com/44798353/102296299-052efe00-3f88-11eb-984f-fab58ebb58b8.png)

You will find that there are multiple descriptions of Angular Material, In order to avoid confusion, I have merged all the instructions for upgrading Angular Material into one description.

![image](https://user-images.githubusercontent.com/44798353/102296576-969e7000-3f88-11eb-8d7c-9fdcf9d24a7f.png)

Reply:
I see. 英文版没有, 我才问。